torch/utils/cpp_extension.py

```diff
@@ -719,6 +720,10 @@ def load_inline(name,
             automatically determined based on whether ``cuda_sources`` is
             provided. Set it to `True`` to force CUDA headers
             and libraries to be included.
+        with_pytorch_error_handling: Determines whether pytorch error and
+            warning macros are handled by pytorch instead of pybind. This
+            requires an intermediary function and should be deactivated
+            if this intermediary function cause issues.
```

Comment:
I'm not sure, as a user, what I'm supposed to do with this information. What does an "intermediary function" do? How can I tell if it's causing issues? If we want to let users debug problems, it would be better to tell them all the gory details of how our binding code is generated, so they can make an informed decision here.